irr/src/tests/CMakeLists.txt

```diff
@@ -0,0 +1,33 @@
+add_executable(tests
+	testCGLTFMeshFileLoader.cpp
+	"${PROJECT_SOURCE_DIR}/source/Irrlicht/CReadFile.cpp"
+)
+
+set_target_properties(tests PROPERTIES
+	CXX_STANDARD 11
+	CXX_STANDARD_REQUIRED YES
+	CXX_EXTENSIONS NO
+)
+
+target_compile_options(tests
+	PRIVATE
+		"$<$<CXX_COMPILER_ID:GNU>:-Wall>"
```

Comment:
Why not enable warnings on Clang and MSVC?

Reply:
tbh I expect this entire file to become pretty much obsolete at some point